install/nvidia

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
Should be `install/nvidia.sh` like the others and then we can do the conditional in here as well.

Reply:
I'd rather we just gather everything related to the nvidia install, including the conditional whether to do it or not, inside this install/nvidia.sh. Then it's entirely self contained 👍